test/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Tests' output files
+_out*.cfg
```

Comment:
Pedantic, but slightly more specific according to the naming scheme.
```suggestion
_out_*.cfg
```

Reply:
Probably a good idea, yeah

Reply:
Resolved in 46b17a61d322984d5e3cbe61f85652b151d414c7